modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -3836,7 +3870,7 @@
     end
   end
 
-  describe 'POST #submit_form_526 not using md5 lookup' do
+  describe 'POST #submit not using md5 lookup' do
```

Comment:
Another typo fix, `submit_form_526` is in v1, not v2 this was likely a copy & paste mistake on my part originally.